scheduler/plugin/mask.go

```diff
@@ -49,6 +49,7 @@ func maskInterfaces(plugin *wasmPlugin) (framework.Plugin, error) {
 		iPreBindPlugin |
 		iPostBindPlugin)
 
+	// TODO: ask about PostFilter
```

Comment:
How should I add `iPostFilterPlugin` for this `switch-case` logic?

Reply:
No need to add.
When filter is used, postfilter is automatically enabled. I cannot think of any usecase where postfilter needs to be used without filter for now.

https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/2275d37d42e823a564f28bb41f6e9bc3daa3ec88/scheduler/plugin/mask.go#L241-L246